src/simple_cache_client.rs

```diff
@@ -1873,79 +1874,20 @@ impl SimpleCacheClient {
     ///
     /// # Arguments
     ///
-    /// * `cache_name` - name of cache.
-    /// * `set_name` - name of the set.
-    /// * `order` - specify ascending or descending order
-    /// * `limit` - optionally limit the number of results returned
-    /// * `range` - constrain to a range of elements by index or by score
-    ///
-    /// # Example
-    /// ```
-    /// # fn main() -> momento_test_util::DoctestResult {
-    /// # momento_test_util::doctest(|cache_name, auth_token| async move {
-    /// use std::time::Duration;
-    /// use momento::SimpleCacheClientBuilder;
-    /// use momento::sorted_set::Order;
-    ///
-    /// let mut momento = SimpleCacheClientBuilder::new(auth_token, Duration::from_secs(30))?
-    ///     .build();
-    ///
-    /// match momento.sorted_set_fetch(
-    ///     &cache_name,
-    ///     "test sorted set",
-    ///     Order::Ascending,
-    ///     None,
-    ///     None
-    /// ).await?.value {
-    ///     Some(set) => {
-    ///         println!("sorted set elements: (score and name)");
-    ///         for entry in &set {
-    ///             println!("{} {:?}", entry.score, entry.name);
-    ///         }
-    ///     },
-    ///     None => println!("sorted set not found!"),
-    /// }
-    /// # Ok(())
-    /// # })
-    /// # }
-    /// ```
+    /// * `_cache_name` - name of cache.
+    /// * `_set_name` - name of the set.
+    /// * `_order` - specify ascending or descending order
+    /// * `_limit` - optionally limit the number of results returned
+    /// * `_range` - constrain to a range of elements by index or by score
     pub async fn sorted_set_fetch(
         &mut self,
-        cache_name: &str,
-        set_name: impl IntoBytes,
-        order: sorted_set::Order,
-        limit: impl Into<Option<NonZeroU32>>,
-        range: Option<sorted_set::Range>,
+        _cache_name: &str,
+        _set_name: impl IntoBytes,
+        _order: sorted_set::Order,
+        _limit: impl Into<Option<NonZeroU32>>,
+        _range: Option<sorted_set::Range>,
```

Comment:
what's up with these name changes?  This is a public API, right?

Reply:
since this function is now empty, these variables are not used. Clippy wants unused variables to start with `_`

Reply:
yeah i get it now.  it's fine pending resolution of the sortedset convo below.